README.md

```diff
@@ -1 +1,57 @@
-[![Hits](https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2Fdsc-jiit-128%2FDSC-site&count_bg=%233D58C8&title_bg=%23555555&icon=&icon_color=%23E7E7E7&title=smash&edge_flat=false)](https://hits.seeyoufarm.com)
+
+![Logo](https://media-exp1.licdn.com/dms/image/C4E1BAQEeEW6C2U7kNg/company-background_10000/0/1570735746624?e=2159024400&v=beta&t=gZJK_A_8YqAW9SoU9FLkheaz8_E0Is9A2uk0cYEqdNI)
+
+    
+# DSC-site
+
+DSC(Developers Student Club) - JIIT128 is a Google Developers program aimed for college students to help them build their mobile, web, machine learning, cloud skills, etc
+
+
+## Installation
+
+Installing React
+```
+    npm install -g react-cli react
+    npm install -g create-react-app
+    npx create-react-app my-app
+    cd my-app
+    npm start
+
+    //Is used to update the version only such as 16.0.0 can be update to latest 16.13.0 but.. 
+    //15.0.0 can't be updated to 16.13.0 or to 16.0.0 with this
+    npm update react
+
+    //Use this to go from 15.0.0 to 16.13.0
+    npm install --save react@latest
+    npm i react
+```
+
+Making of react app
+
+```
+    npx create-react-app <appname>  //  eg:  npx create-react-app blog
+    cd <appname>  // cd blog
+    npm start
+```
+
+
+
+## Run Locally
+
+Clone the project
+
+```bash
+  git clone https://github.com/dsc-jiit-128/DSC-site.git
+```
+
+Go to the project directory
+
+```bash
+  cd DSC-site
+```
+
+  
+## 🚀 About Us
+We are basically a community with connections all over the world. Open to any student, ranging from novice developers who are just starting, to advanced developers who want to further enhance their skills, we provide a platform for students to learn and collaborate in solving problems around them with the help of technology.

```

Comment:
DSC(Developers Student Club) - JIIT128 is a Google Developers program aimed at college students to help them build their mobile, web, machine learning, cloud skills, etc. We are basically a community with connections all over the world. Open to any student, ranging from novice developers who are just starting, to advanced developers who want to further enhance their skills, we provide a platform for students to learn and collaborate in solving problems around them with the help of technology.

Reply:
ok sir